packages/contextual-help/src/spectrum-contextual-help.css

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
+
+:host {
+    --spectrum-contextual-help-padding: var(--spectrum-spacing-400);
+    --spectrum-contextual-help-content-spacing: var(--spectrum-spacing-100);
+    --spectrum-contextual-help-link-spacing: var(--spectrum-spacing-300);
+    --spectrum-contextual-help-heading-size: var(
+        --spectrum-contextual-help-title-size
+    );
+    --spectrum-contextual-help-heading-color: var(--spectrum-heading-color);
+    --spectrum-contextual-help-body-color: var(--spectrum-body-color);
+}
+
+.spectrum--large .popover {
```

Comment:
This seems like it doesn't belong... It _should_ be processed out on it's own, but it may exist outside of our current CSS processing. Can you confirm the selector in the origin CSS?

Reply:
Here it is: https://github.com/adobe/spectrum-css/blob/main/components/contextualhelp/index.css#L25 I can not find other components with this `spectrum--large` class. Maybe the `--spectrum-contextual-help-content-spacing` variable should be moved in the `large-vars` and `medium-vars`?

Reply:
@Rocss Can you please confirm this with @pfulton

Reply:
@Rajdeepc  Since I'm surfacing a dialog on mobile, can we exclude this CSS?

Reply:
Yes!